File: 75-sort-colors/75. Sort Colors.java
class Solution {
    public void sortColors(int[] nums) {

        int n=nums.length;
        int low=0,mid=0,high=n-1;
        while(mid<=high){
            if(nums[mid]==0){
                int temp=nums[low];
                nums[low]=nums[mid];
                nums[mid]=temp;
                mid+=1;
                low+=1;
            }
            else if(nums[mid]==1){
                mid+=1;
            }
            else{
                int temp=nums[high];
                nums[high]=nums[mid];
                nums[mid]=temp;
                high-=1;
            }
        }
    }
}

File: 75-sort-colors/75. Sort Colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        l = 0
        r = len(nums)-1
        m = 0
        while(m<=r):
            if(nums[m]==0):
                nums[m],nums[l]= nums[l], nums[m]
                m+=1
                l+=1
            elif(nums[m]==1):
                m+=1
            else:
                nums[m],nums[r]= nums[r], nums[m]
                r-=1


        """
        Do not return anything, modify nums in-place instead.
        """
        